test/e2e/run_networking_test.go

```diff
@@ -24,6 +24,7 @@ var _ = Describe("Podman run networking", func() {
 	hostname, _ := os.Hostname()
 
 	It("podman verify network scoped DNS server and also verify updating network dns server", func() {
+		Skip("FIXME: needs netavark > 1.10.2, available >= 2024-02-02")
```

Comment:
I will keep trying to rebuild CI VMs, and once I get a working netavark, will remove these skips.